index.bs

```diff
@@ -1194,6 +1194,15 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
                         :   {{AuthenticatorAttestationResponse/attestationObject}}
                         ::  |attestationObject|
 
+                        :   {{AuthenticatorAttestationResponse/[[transports]]}}
+                        ::  A sequence of zero or more unique {{AuthenticatorTransport}} values, in lexicographical order, that the |authenticator| is believed to support.
+
+                            If a user agent does not wish to divulge this information it MAY substitute an arbitrary sequence designed to preserve privacy. This sequence MUST still be valid, i.e. lexicographically sorted and free of duplicates. For example, it may use the empty sequence. Either way, in this case the user agent takes the risk that [=[RP]=] behavior may be suboptimal.
```

Comment:
Is it still valuable to allow the client to return an arbitrary bogus sequence? Why not require the client to return either the true sequence, or an empty sequence if it wishes to keep the true sequence private?

Reply:
So the edit I'm suggesting is to change this:

>If a user agent does not wish to divulge this information it MAY substitute an arbitrary sequence designed to preserve privacy. This sequence MUST still be valid, i.e. lexicographically sorted and free of duplicates. For example, it may use the empty sequence. Either way, in this case the user agent takes the risk that [=[RP]=] behavior may be suboptimal. 

to this:

>If a user agent does not wish to divulge this information it MAY set this to the empty sequence. In this case the user agent takes the risk that [=[RP]=] behavior may be suboptimal.

Reply:
If the client wishes to blend into the crowd, it would seem best to give it flexibility? Indeed, even if the spec says that the values must either be completely real or else empty, I expect that clients that wished to maintain privacy would probably do whatever they thought least conspicuous and most compatible irrespective of what the spec says.

I think @jcjones is the closest that we have to a rep for Tor Browser here, and I'll defer to him on this point if he has an opinion, but I'm not yet convinced about tightening this.

Reply:
Tor Browser historically chooses a value that will not cause breakage but allows them to look like the most popular <thing> in question. (E.g. choosing a version of the OS that is most popular.)  So I agree that if the spec requires either 'completely real or empty' that would probably be ignored.

Reply:
Ok, that's enough for me. :)